internal/discord/command-handlers.go

```diff
@@ -713,3 +729,69 @@ func (d *Discord) InteractionCreate(s *discordgo.Session, i *discordgo.Interacti
 		d.ShowAllStrikes(s, i)
 	}
 }
+
+// RoleRemoveAddHelper removes a role `roleIDToRemove` and then adds a role `roleIDToAdd` to the user `userID`
+func (d *Discord) RoleRemoveAddHelper(state *InteractionState, userID string, roleToRemove string, roleToAdd string) error {
+	roleIDToRemove := d.Roles[state.interaction.GuildID][roleToRemove].ID
+	roleIDToAdd := d.Roles[state.interaction.GuildID][roleToAdd].ID
+
+	// Attempt to remove role first
+	err := state.session.GuildMemberRoleRemove(state.interaction.GuildID, userID, roleIDToRemove)
+	if err != nil {
+		// Abort entire process if role removal fails
+		tempstr := fmt.Sprintf("Could not remove the role <@&%v> from user <@%v>", roleIDToAdd, userID)
+		fmt.Printf("%v: %v\n", tempstr, err)
+		RespondAndAppendLog(state, tempstr)
+		return err
+	} else {
+		// Otherwise add role
+		err = state.session.GuildMemberRoleAdd(state.interaction.GuildID, userID, roleIDToAdd)
+		if err != nil {
+			tempstr := fmt.Sprintf("Could not give user <@%v> role <@&%v>", userID, roleIDToAdd)
+			fmt.Printf("%v: %v\n", tempstr, err)
+			RespondAndAppendLog(state, tempstr)
+			return err
+		} else {
+			return nil
+		}
+	}
+}
+
+func (d *Discord) ExileUser(state *InteractionState, userID string, reason string) error {
+	// Check user for specified roles
+	roleToRemove := "Verified"
+	roleToAdd := "Exiled"
```

Comment:
ehhhh we should probably make an issue since we want this in PR to go in soon, but we should probably move this to the constants file